memory/homenode.py.data/homenode.c

```diff
@@ -31,11 +31,12 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-
 #define errmsg(x, ...) fprintf(stderr, x, ##__VA_ARGS__),exit(1)
 #define __NR_HOME_NODE 450
 
-
+#ifndef MPOL_PREFERRED_MANY
+#define MPOL_PREFERRED_MANY	5
+#endif
```

Comment:
Any particular reason why MPOL_PREFERRED_MANY is separately defined but MPOL_BIND is not?

Reply:
MPOL_PREFERRED_MANY was added to numaif.h by this commit :
https://github.com/numactl/numactl/commit/d538c6453618f3dd6445056a9a343f52fdb23c10#diff-010b924e5fd621741c7e4b7d7461a08c756c71547179d79a5cfdf53fe256b385
merged into master :   v2.0.16

For distros where the above patch is not present, I am defining the flag explicitly.